### DISCUSSION TITLE
Shall we have a constants module?

### DISCUSSION THREAD

**Discussion by BorjaEst** [imk-toolkit/imk-toolkit#66] [Ideas]
I see that the method [num_den](https://github.com/imk-toolkit/imk-toolkit/blob/master/imktk/dataarray_methods/num_den.py) defines and assigns its constants inside the script. 

In long term, having a common module with all constants might simplify code and reduce errors.

**Comment by ucyo** [comment]
There is already a discussion about this at #47. Closing the discussion, here.
